lib/slide_rule/distance_calculator.rb

```diff
@@ -72,7 +73,7 @@ def calculate_weighted_distances(i1, i2)
       distances = @rules.map do |attribute, rule|
```

Comment:
I think we should rename the hash to options everywhere else. I think that would make it clear what it is referring to.